src/main/java/com/amazonaws/services/sqs/AmazonSQSIdleQueueDeletingClient.java

```diff
@@ -278,7 +296,7 @@ private void heartbeatToQueueIfNecessary(String queueUrl) {
         QueueMetadata queueMetadata = queues.get(queueUrl);
         if (queueMetadata != null) {
             Long lastHeartbeat = queueMetadata.heartbeatTimestamp;
-            if (lastHeartbeat == null || (System.currentTimeMillis() - lastHeartbeat) > 2 * HEARTBEAT_INTERVAL_SECONDS) {
+            if (lastHeartbeat == null || (System.currentTimeMillis() - lastHeartbeat) > 2 * heartbeatIntervalSeconds) {
```

Comment:
I just noticed there's an existing bug here, comparing milliseconds on the LHS to seconds on the right. Let's log a separate issue for that - reveals a testing gap as well.

Reply:
Added separate issue concerning this